ParisBNF/abb/BNFabb92.xml

```diff
@@ -0,0 +1,682 @@
+                                        ኵሎ፡ አሚረ፡ ዘአስተርአየ፡ አሚሃ፡ ወበኵሉ፡ መዋዕል፡ ኢይትከሀል። በከመ፡ ባሕር፡ ኢይትኋለቍ፡
+                                        መዋግዲሁ፡ ወኢየኅልቅ፡ ወኢይትፈጸም፡ ከማሁ፡ ተአምሪሁ፡ ለጊዮርጊስ፡ መኑ፡ ይከል፡ ይንግር፡
+                                        ወይኆለቍ፡ ኵሎ። ወበውእቱ፡ መዋዕል፡ ፀብኡ፡ ዕልዋን፡ ለብሔረ፡ ቢላቆንያ። </incipit>
+                                </msItem>
+
+                            </msItem>
+                            <msItem xml:id="ms_i6">
+                                <locus from="58ra" to="66vb"/>
+                                <title type="complete" xml:lang="en" ref="LIT6171HolyCross">The
+                                    invention of the Holy Cross by the empress Helena (<date
+                                        ref="ethiocal:Hedar8">8th of Ḫǝdār</date>)</title>
+                                <incipit xml:lang="gez"><hi rend="rubric">ነገረ፡ አስተርእዮተ፡</hi> መስቀሉ፡
+                                    ዐቢይ፡ ማሕየ<hi rend="rubric">ዊ፡ ለእግዚእነ፡ ወአምላ</hi>ክነ፡ ኢየሱስ፡ ክርስቶስ፡
+                                        <hi rend="rubric">አመ፡ ፰ ለኅዳር፡ በመዋዕ</hi>ለ፡ ቈስጠንጢኖስ፡ ንጉሥ፡ <hi
+                                        rend="rubric">ዐቢይ፡ መሀይምን፡ ኄር፡</hi></incipit>
+
+                            </msItem>
+                            <msItem xml:id="ms_i7">
+                                <locus from="67ra" to="76vb"/>
+                                <title type="complete" xml:lang="en" ref="LIT1651Homily">Homily of
+                                        <persName ref="PRS5720JohnChr">John Chrysostom</persName> on
+                                    the four beasts (<date ref="ethiocal:Hedar8">8th of
+                                    Ḫǝdār</date>)</title>
+                                <incipit xml:lang="gez"><hi rend="rubric">ድርሳን፡ ዘብፁዕ፡ ወቅ</hi>ዱስ፡
+                                    ዮሐንስ፡ ሊቀ፡ ጳጳሳት፡ <hi rend="rubric">ዘቈስጠንጥንያ፡ በእንተ፡ ፬፡ </hi>እንስሳ፡
+                                    ዐቢያት፡ ወመን<hi rend="rubric">ፈሳዊያን፡ አመ፡ ፰ ለኅዳር፡ </hi>ዘይትነበብ፡ በሰላመ፡
+                                    እግዚአብሔር፡ አሜን።</incipit>
+                            </msItem>
+                            <msItem xml:id="ms_i8">
+                                <locus from="77ra" to="110va"/>
+                                <title type="complete" xml:lang="en" ref="LIT6174Severus318"
+                                    >Treatise of <persName ref="PRS8788Severos">Severus (of
+                                        Ašmunayn)</persName> on the Council of the 318 Fathers (at
+                                    Nicaea)</title>
+                                <note>In the upper margin, there is a reading note that names this
+                                    text <foreign xml:lang="gez">ዝክረ፡ ማኅበር፡</foreign> and suggests
+                                    to read it on <date ref="ethiocal:ethiocal:Hedar9">9th of
+                                        Ḫǝdār</date>. </note>
+                                <note> While cataloguing this manuscript, <persName
```

Comment:
Is the information pertinent to the manuscript or the work? It seems the latter to me